tests/test_class.cpp

```diff
@@ -434,8 +434,7 @@ TEST_SUBMODULE(class_, m) {
     struct SamePointer {};
     static SamePointer samePointer;
     py::class_<SamePointer, std::unique_ptr<SamePointer, py::nodelete>>(m, "SamePointer")
-        .def(py::init([]() { return &samePointer; }))
-        .def("__del__", [](SamePointer&) { py::print("__del__ called"); });
```

Comment:
If this looks like a "sweep it under the carpet" kind of thing, you wouldn't be wrong. But we were never really using the fact that `__del__` prints something anyway (possibly because it already didn't work on PyPy?).